tests/test_data_store_api_spatialite.py

```diff
@@ -711,7 +711,7 @@ def test_new_comment_created_successfully(self):
                 self.sensor, self.current_time, "Comment", self.comment_type,
             )
 
-            # there must be no entry because it's kept in-memory
+            # there must be noget_plat entry because it's kept in-memory
```

Comment:
typo?

Reply:
I think you're looking at the outdated file because it was the typo. I changed it to "no" again.